src/views/Home/Home.test.jsx

```diff
@@ -10,6 +12,29 @@ const user = {
   color: 'crimson',
 }
 
-test('Should render the user profile', () => {
+test('Should render the user profile', async () => {
+  render(<Home user={user}/>)
+  screen.debug();
+
+const name = screen.getByText('Vonta');
+expect(name).toHaveTextContent('Vonta');
+
+const saying = screen.getByText('Res Non Verba');
+expect(saying).toHaveTextContent('Res Non Verba');
+
+screen.findByAltText('avatar');
+// expect(image).toHaveTextContent('avatar');
+
+screen.findByAltText('header');
+
+screen.findByText('Interests');
+
+
+
+
+user.likes.forEach(item=> screen.getByText(`${item}`));
+
```

Comment:
Remember to add in variable names so that we can test for them to be in the document. We can use expect(variable).toBeInTheDocument()